pkg/orchestrator/model/types.go

```diff
@@ -77,6 +77,8 @@ const (
 	K8sVerticalPodAutoscaler = 22
 	// K8sHorizontalPodAutoscaler represents a Kubernetes Horizontal Pod Autoscaler
 	K8sHorizontalPodAutoscaler = 23
+	// ECSTask represents an ECS Task
+	ECSTask = 24
```

Comment:
should we change this to not disrupt the k8s resource increment?
maybe start in another range (eg 100?)

Reply:
now it's 150 https://github.com/DataDog/datadog-agent/blob/7bac94cf2d33a37f20e5c7499c742b1d499fa584/pkg/orchestrator/model/types.go#L81
,matching with https://github.com/DataDog/agent-payload/pull/281/files#diff-69b34851fbf6263111121260c46c57084a1fe6e34e9b750a5a673694f2dfe1b0R768